packages/realm-web/package.json

```diff
@@ -12,15 +12,31 @@
   },
   "scripts": {
     "prepack": "npm run build",
-    "prepare": "npm run update-types",
-    "update-types": "ts-node --project scripts/tsconfig.json scripts/update-types.ts",
-    "generate-types": "tsc --project src/dom/tsconfig.json --declaration --emitDeclarationOnly --declarationDir types/generated",
-    "build": "npm run generate-types && rollup --config",
+    "build": "wireit",
     "start": "npm run build -- --watch",
     "lint": "eslint --ext .js,.ts .",
     "test": "mocha 'src/tests/**/*.test.ts'",
     "postversion": "ts-node --project scripts/tsconfig.json scripts/postversion.ts"
   },
+  "wireit": {
+    "build": {
+      "command": "rollup --config",
+      "dependencies": ["update-types", "generate-types"],
+      "files": ["rollup.config.mjs", "src/**/*.ts", "src/**/tsconfig.json", "tsconfig.json", "types"],
+      "output": ["dist"]
+    },
+    "generate-types": {
+      "command": "tsc --project src/dom/tsconfig.json --declaration --emitDeclarationOnly --declarationDir types/generated",
+      "dependencies": ["update-types"],
+      "files": ["src/**/*.ts", "src/**/tsconfig.json", "tsconfig.json"],
+      "output": ["types/generated"]
+    },
+    "update-types": {
```

Comment:
Should this have a dependency on `package/realm` types being generated?

Reply:
The `update-types` script is copying the files from the `./types` of the root directory, so these are static files that isn't generated. This needs to be fixed at some point (as we want those files gone), but I'd say we should probably wait for that until we know the future of the WASM binding.